platform/openide.util/test/unit/src/org/openide/util/BaseUtilitiesTest.java

```diff
@@ -309,75 +303,73 @@ public void test_toURI_producesURI_wchichResolvesAndWrapsProperly()
         URI wrappedResolvedURI = new URI("jar:" + resolvedURI + "!/");
         assertEquals(wrappedURI, wrappedResolvedURI);
     }
+    
+    //--------------------------------------------------------------------------
+    public void test_parseParameters_throwsNullPointer_whenGivenNullArgument() {
+
+        expectNullPointer(() -> parseParameters(null));
+    }
+
+    //--------------------------------------------------------------------------
+    public void test_parseParameters_returnsEmptyArray_whenGivenEmptyString() {
+
+        assertEquals(0, parseParameters("").length);
+        assertEquals(0, parseParameters("   \t\r\n").length);
+    }
+
+    //--------------------------------------------------------------------------
+    public void test_parseParameters_returnsProperArray_forProperInput() {
+
+        String[] args;
```

Comment:
Why define the variable in a different line ?

Reply:
so that we can reuse it many times.

Reply:
Isn't required in Java. You can inline variable declaration with definition

Reply:
But then I'd have to create a new variable every time (like args1, args2, ....). Seems not worth it in this case.

some rules of thumb applicable to production code can bee loosened in case of test code for practical purposes

Reply:
ohh. sorry - you meant "the first time the variable has been used. 
I just wanted the line 325 to be shorter